internal/server/place_stat_vars.go

```diff
@@ -135,6 +135,21 @@ func (s *Server) GetPlaceStatVarsUnion(
 func (s *Server) GetPlaceStatVarsUnionV1(
 	ctx context.Context, in *pb.GetPlaceStatVarsUnionRequest) (
 	*pb.GetPlaceStatVarsUnionResponseV1, error) {
+	statVars := in.GetStatVars()
+	dcids := in.GetDcids()
+	if len(statVars) > 0 && len(dcids) > 0 {
```

Comment:
Can you add a comment for why using separate cache here.

Reply:
done